font.js

```diff
@@ -1,20 +1,21 @@
 var WebFont = require("webfont");
 
 exports.locate = function(load) {
-  return load.name;
+  return decodeURI(load.name).match(/[^/]*$/)[0];
```

Comment:
I think it will, since it looks like the entire font specifier got URI encoded. But maybe your right, and it would be sad if it didn't work.

I'm not sure if there are any, but if there is just one (or if one gets added) it should be usable by this library.